### ISSUE TITLE
Assets Not Downloading

### ISSUE THREAD

**Issue by Switchfoot55** [bmaltais/kohya_ss#525]
I've attempted several un/reinstalls the last few days, but continue to run into this error. I tried running the Dreambooth Lora with a different Stable Diffusion model and it started to download the assets again, but then errored out. Is there a way to manually download these? 

OR, if there some other underlying issue I'm not seeing?

text_encoder\model.safetensors not found
Downloading (…)rocessor_config.json: 100%|█████████████████████████████████████████████| 342/342 [00:00<00:00, 172kB/s]
Downloading (…)cheduler_config.json: 100%|█████████████████████████████████████████████| 345/345 [00:00<00:00, 115kB/s]
Downloading (…)_encoder/config.json: 100%|████████████████████████████████████████████| 633/633 [00:00<00:00, 79.4kB/s]
Downloading (…)cial_tokens_map.json: 100%|████████████████████████████████████████████| 460/460 [00:00<00:00, 92.3kB/s]
Downloading (…)okenizer_config.json: 100%|█████████████████████████████████████████████| 824/824 [00:00<00:00, 275kB/s]
Downloading (…)b1b/unet/config.json: 100%|█████████████████████████████████████████████| 909/909 [00:00<00:00, 456kB/s]
Downloading (…)3b1b/vae/config.json: 100%|█████████████████████████████████████████████| 611/611 [00:00<00:00, 204kB/s]
Downloading (…)tokenizer/merges.txt: 100%|██████████████████████████████████████████| 525k/525k [00:09<00:00, 55.0kB/s]
Downloading (…)tokenizer/vocab.json: 100%|████████████████████████████████████████| 1.06M/1.06M [00:20<00:00, 51.7kB/s]
Downloading (…)"model.safetensors";:   1%|▎                                    | 10.5M/1.36G [09:39<20:43:37, 18.1kB/s]
Downloading (…)_pytorch_model.bin";:   3%|█▏                                     | 10.5M/335M [09:36<4:56:51, 18.2kB/s]
Downloading (…)"pytorch_model.bin";:   1%|▎                                    | 10.5M/1.36G [09:39<20:42:58, 18.1kB/s]
Downloading (…)_model.safetensors";:   0%|                                     | 10.5M/3.46G [09:38<52:48:22, 18.2kB/s]
Downloading (…)_model.safetensors";:   3%|█▏                                     | 10.5M/335M [09:36<4:56:50, 18.2kB/s]
Downloading (…)_pytorch_model.bin";:   0%|                                     | 10.5M/3.46G [09:38<52:49:19, 18.2kB/s]
Fetching 16 files: 100%|███████████████████████████████████████████████████████████████| 16/16 [09:40<00:00, 36.27s/it]
D:\SD\Kohya\kohya_ss\venv\lib\site-packages\transformers\models\clip\feature_extraction_clip.py:28: FutureWarning: The class CLIPFeatureExtractor is deprecated and will be removed in version 5 of Transformers. Please use CLIPImageProcessor instead.
  warnings.warn(
Traceback (most recent call last):
  File "D:\SD\Kohya\kohya_ss\train_network.py", line 724, in <module>
    train(args)
  File "D:\SD\Kohya\kohya_ss\train_network.py", line 135, in train
    text_encoder, vae, unet, _ = train_util.load_target_model(
  File "D:\SD\Kohya\kohya_ss\library\train_util.py", line 2654, in load_target_model
    pipe = StableDiffusionPipeline.from_pretrained(name_or_path, tokenizer=None, safety_checker=None)
  File "D:\SD\Kohya\kohya_ss\venv\lib\site-packages\diffusers\pipeline_utils.py", line 709, in from_pretrained
    loaded_sub_model = load_method(os.path.join(cached_folder, name), **loading_kwargs)
  File "D:\SD\Kohya\kohya_ss\venv\lib\site-packages\transformers\modeling_utils.py", line 2301, in from_pretrained
    state_dict = load_state_dict(resolved_archive_file)
  File "D:\SD\Kohya\kohya_ss\venv\lib\site-packages\transformers\modeling_utils.py", line 413, in load_state_dict
    return safe_load_file(checkpoint_file)
  File "D:\SD\Kohya\kohya_ss\venv\lib\site-packages\safetensors\torch.py", line 100, in load_file
    result[k] = f.get_tensor(k)
RuntimeError: self.size(-1) must be divisible by 4 to view Byte as Float (different element sizes), but got 10130705
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2800.0_x64__qbz5n2kfra8p0\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2800.0_x64__qbz5n2kfra8p0\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "D:\SD\Kohya\kohya_ss\venv\Scripts\accelerate.exe\__main__.py", line 7, in <module>
  File "D:\SD\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\accelerate_cli.py", line 45, in main
    args.func(args)
  File "D:\SD\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\launch.py", line 1104, in launch_command
    simple_launcher(args)
  File "D:\SD\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\launch.py", line 567, in simple_launcher
    raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)
subprocess.CalledProcessError: Command '['D:\\SD\\Kohya\\kohya_ss\\venv\\Scripts\\python.exe', 'train_network.py', '--v2', '--v_parameterization', '--enable_bucket', '--pretrained_model_name_or_path=stabilityai/stable-diffusion-2', '--train_data_dir=D:/SD/GUI/Data/Set 2/Crissy lora/image', '--resolution=512,512', '--output_dir=D:/SD/GUI/Data/Set 2/Crissy lora/model', '--logging_dir=D:/SD/GUI/Data/Set 2/Crissy lora/log', '--network_alpha=128', '--save_model_as=safetensors', '--network_module=networks.lora', '--text_encoder_lr=5e-5', '--unet_lr=0.0001', '--network_dim=128', '--output_name=Crissy', '--lr_scheduler_num_cycles=1', '--learning_rate=0.0001', '--lr_scheduler=constant', '--train_batch_size=1', '--max_train_steps=1500', '--save_every_n_epochs=1', '--mixed_precision=fp16', '--save_precision=fp16', '--seed=1234', '--caption_extension=.txt', '--cache_latents', '--optimizer_type=AdamW8bit', '--max_data_loader_n_workers=1', '--clip_skip=2', '--bucket_reso_steps=64', '--mem_eff_attn', '--gradient_checkpointing', '--xformers', '--bucket_no_upscale']' returned non-zero exit status 1.

If I do another un/reinstall, should I track down the assets, delete them, and then reinstall?

**Comment by bmaltais** [comment]
I think your version of python might be part of the issue. I see you are running 3/10/3 but 3.10.9 is what I am testing with.

Perhaps you can try to upgrade the python ver sion, delete the full kohya_ss folder and then attempt a fresh install?

**Comment by Switchfoot55** [comment]
when I run python -V it shows Python 3.10.10, so I think I'm all the way up to date? 

Can anyone confirm if the assets listed above that did not download are available elsewhere to manually update?

**Comment by Switchfoot55** [comment]
For additional context, I've been trying to run the training on the v1.5 Source Model. The first time I attempted this, I got a similar error. After multiple un/reinstalls, I decided to try a different source model. That is when the downloads started to happen. However, my internet connection was interrupted and it errored out. I attempted again on that source model only to encounter an error immediately. Thus far, each new source model attempt has started with the downloads listed above. But, in each attempt, my internet connection was interrupted. 

I am making a final attempt to see if that will solve it. Where would these downloads be going? And, to echo above, is there a way to manually push these through? If these are in fact the issue, then my hope is I can get the assets on the system and rerun the training in the v1.5 Source Model.
